### DISCUSSION TITLE
122

### DISCUSSION THREAD

**Discussion by giscus** [CODe5753/giscus#4] [Comments]
# 122

서론 후회된다. 과거로 돌아가고 싶다. 후회를 언제 하시나요? "후회된다"라는 표현을 언제 쓰시나요? 저는 20대 초반, 어느샌가부터 "후회"라는 단어를 떠올리지 않게 되었는데요. 매 선택의 순간에서 후회하지 않는 선택을 했고, 그 선택 이후에 후회되지 않도록 살아갔기 때문이었던 것 같습니다. 이번 아티클에서는 신입 개발자로서의 2년을 돌아보며 한순간도 후회되지 않았던, 과거로 돌아가라고 해도 돌아가지 않겠다고 확신할 수 있었던 이유에 대해서 고민해 보고 매 순간 어떤 태도로 상황을 대했는지 이야기해보겠습니다. 동기부여하기 2년 전 첫 선택은 넥슨코리아 신입 공채 최종합격 발표 후였습니다. 한 컨설턴트님께서 다시 한번 생각해 보라며 승현님이 기대하는 트래픽도 없을 것이고 게임 회사이기 때문에 상대적으로 ..

https://imksh.com/122

**Comment by devFancy** [comment]
좋은 글 감사합니다! 👍

지난해 저는 팀원들과 8개월 동안 사이드 프로젝트를 진행했지만, 결국 실패를 맛봤습니다. 저 역시 '실패'의 기준을 승현님과 동일하게 생각했어요. 이러한 실패를 극복하고, 이전의 프로젝트를 개선하고자 저 혼자서 version2로 사이드 프로젝트를 진행하면서 version1에 겪었던 문제를 해결했어요. 

실패는 미래의 '나'를 성장시키는 요인인 것 같다는 생각이 드네요.

**Reply by CODe5753** [reply]
@devFancy 님 안녕하세요
프로젝트 개선을 위해 혼자 구현 하겠다는 결정과 과정이 결코 쉽지 않았을텐데 정말 대단하십니다 👍🏻
저는 그러지 못했는데, 열정이 가득한 모습을 보고 반성하게 되네요 ㅎㅎ

말씀하신대로 **실패는 나를 성장시킬 수 있는 기회를 제공하지만 그 실패에 어떻게 대처하느냐에 따라 그 기회를 잡을 수도, 놓칠 수도 있겠네요.**
어쩌면 저는 아직 그 기회가 떠나지 않았는데 잡지도, 놓지도 않은 방관하고 있는 상황일지도 모르겠군요 🥹

좋은 인사이트 감사합니다 ☺️
앞으로도 열정 가득한 모습 기대할게요!! 🙌🏻

**Comment by Denia-park** [comment]
계속해서 더 나아지기 위해 열심히 노력하신게 느껴지네요.
동기부여 받고 갑니다 ㅎㅎ

설날이 지나버렸지만 새해 복 많이 받으세요.

**Reply by CODe5753** [reply]
@Denia-park 님 안녕하세요
동기부여가 되셨다니 영광입니다 🙇🏻‍♂️
이 댓글을 쓰는 시점은 3월을 앞두고 있는데, 
새해 복 많이 받으시고 따뜻한 봄을 맞이하셔요!
감사합니다 ☺️

**Comment by dlwlxns4** [comment]
혼자서 이악물고 해내는 것이 능사가 아니라 주변 동료에게 도움을 요청하는 것도 능력이다
저도 이와 유사한 말 듣고 정말 충격에 많이빠졌었어요. 

2년경력 후 원하는 게임 만들고싶어서 퇴사 후 사이드 프로젝트하다 실패 후 번아웃 세게겪고 골골거리다 최근에 이직준비를 다시하고있습니다.
약 한달동안 서류합격률이 처참해서 충격받은 찰나 작성해주신 포스트들에서 많이 도움받고있습니다.

포스팅해주신 글들 보며 많은 감정이랑 도움들을 느끼고있는데 정말 좋은 글 감사합니다.

**Reply by CODe5753** [reply]
안녕하세요.  
제가 요새 블로그 관리를 잘 하지 않고 있는데 이렇게 종종 댓글이 달리는 걸 보면 참 신기하네요.  
알람이 올 때마다 '내가 뭘 썼더라..' 하며 한 번씩 보곤 합니다.  
덕분에 잊고 있었던 다짐이나 생각들을 한번 더 되새길 수 있었네요. 감사드립니다.  
이 글을 썼던 당시의 저는 굉장히 열정이 넘치고 호기심도 많았던 시절 같네요.  
아직 뭔가 굳어지지 않은 말랑한 시절 같아서 조금은 쑥스럽기도 합니다.  
  
저는 지금 새로운 환경에서 새로운 사람들, 멋진 사람들을 만나 또 다시 많이 배우고 있습니다.  
여전히 과거에 대한 후회는 하지 않습니다만, 어째서인지 배움에 끝이 없고 아직 너무 부족하다는 사실을 인지할 때마다 무력해지곤 하는데요.  
그러면서도 과거에 제가 이런걸 겪고 다짐해온 것들을 보며 다시 일어나곤 합니다.  

dlwlxns4님 께서도 분명 다시 좋은 흐름을 타실 수 있을 거라 믿습니다.
원하는 게임을 만들고자 했던 그 마음도, 번아웃을 겪고도 다시 도전하고 계신 지금의 모습도 모두 앞으로의 길에 든든한 힘이 되어줄 겁니다.
저 역시 시행착오를 겪을 때마다 '이 경험이 결국엔 나를 단단하게 만들어줄 거다'라고 스스로 다독이곤 했거든요.

앞으로도 과정 속에서 얻는 배움들을 잘 쌓아가시길 응원하겠습니다.
혹여 또 지치실 때에는 예전의 다짐이나 작은 성취들을 떠올리며 힘내셨으면 좋겠어요.
편안한 밤 되세요.
